app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2-pinctrl.dtsi

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Since you're not using any of the pinctrl'd peripherals, this whole file can be skipped.

Reply:
I wondered about needing this. Pinctrls are something I have almost no understanding of and need to learn more about.